tools/istio-iptables/pkg/builder/iptables_builder_impl.go

```diff
@@ -34,22 +34,22 @@ type Rules struct {
 	rulesv6 []*Rule
 }
 
-// IptablesBuilderImpl is an implementation for IptablesBuilder interface
-type IptablesBuilderImpl struct {
+// IptablesBuilder is an implementation for IptablesBuilder interface
```

Comment:
The comments should be changed to `adding iptables rules`

And may rename the file name to iptables_builder.go